src/libs/brand-utils.js

```diff
@@ -0,0 +1,28 @@
+import { brands } from 'src/libs/brands'
+import { getConfig } from 'src/libs/config'
+import * as Utils from 'src/libs/utils'
+
+
+// TODO: Deprecate Datastage (https://broadworkbench.atlassian.net/browse/SATURN-1414)
+export const isAnvil = () => (window.location.hostname === brands.anvil.hostName) || getConfig().isAnvil
+export const isBaseline = () => (window.location.hostname === brands.baseline.hostName) || getConfig().isBaseline
+export const isBioDataCatalyst = () => (window.location.hostname === brands.bioDataCatalyst.hostName) || getConfig().isBioDataCatalyst
+export const isDatastage = () => (window.location.hostname === brands.datastage.hostName) || getConfig().isDatastage
+export const isElwazi = () => (window.location.hostname === brands.elwazi.hostName) || getConfig().isElwazi
+export const isFirecloud = () => (window.location.hostname === brands.firecloud.hostName) || getConfig().isFirecloud
+export const isProjectSingular = () => (window.location.hostname === brands.projectSingular.hostName) || getConfig().isProjectSingular
+export const isRareX = () => (window.location.hostname === brands.rareX.hostName) || getConfig().isRareX
+export const isTerra = () => (window.location.hostname === brands.terra.hostName) || getConfig().isTerra || (!isFirecloud() && !isDatastage() && !isAnvil() && !isBioDataCatalyst() && !isBaseline() && !isElwazi() && !isProjectSingular() && !isRareX())
```

Comment:
Nit: I wonder if it would be cleaner to have these `isX` methods call a common method with `Utils.case` so that they can have a default case for `isTerra`.

Reply:
Or even better if you can eliminate the need for `isX` methods entirely by putting everything into brands.

Reply:
> Nit: I wonder if it would be cleaner to have these `isX` methods call a common method with `Utils.case` so that they can have a default case for `isTerra`.

`() => brands.terra` _is_ the case for defaulting to Terra. Perhaps I'm misunderstanding the ask?

> Or even better if you can eliminate the need for isX methods entirely by putting everything into brands.

That's what I'm hoping to do too but I don't want that work to be part of this PR.

Reply:
I mean that it would be good to avoid the need for this part of the conditional:

```
(!isFirecloud() && !isDatastage() && !isAnvil() && !isBioDataCatalyst() && !isBaseline() && !isElwazi() && !isProjectSingular() && !isRareX())
```

I think a Utils.cond block that deals with all the different types would be able to accomplish this by having `isTerra` be the default condition.

Reply:
Yea I think that could be nicely handled by the further refactoring work I have in mind.